docs/index.html

```diff
@@ -0,0 +1,48 @@
+<form id="myForm">
```

Comment:
You should have all this inside a proper HTML document, e.g.

    <!DOCTYPE html>
    <html>
      <head>
        ..
          <title>My Website</title>
      </head>
      <body>
        .. the rest of your HTML in here ..
      </body>
    </html>